javascript/javascript1/week3/Note.js

```diff
@@ -0,0 +1,63 @@
+const notes = [];
+function addNote( id , content) {
+            
+    if (id === null || isNaN(id)) {
+        return 'Error in ID format' ;
+     }
+     else {
+        notes.push({id,content});
+     }
+       
+
+}
+
+
+function getNoteFromId(id) {
+    
+    console.log(id); //For output reference 
+
+    for (let i = 0; i < notes.length ; i++) {
+        if ( notes[i].id === id) {
+            return notes[i];
+         
+        }
+    }
+             return "not found" ;
+           
+}
+//Get all notes
+function getAllNotes()  {
```

Comment:
actually this will return just ONE of the notes, and not all of them.

Reply:
//Get all notes
function getAllNotes()  {
    for (let i = 0; i < notes.length ; i++) {
        return notes;
    }

 }

Reply:
I realised notes[i] was giving the specific note ,so removed 'i'

Reply:
actually the correct code is:
```function getAllNotes() {
   return notes;
}```

No for loop necessary. Right now your for loop returns after the first iteration, but it doesn't do anything useful.